src/components/Amalgamation/AmalgamatingBusinesses.vue

```diff
@@ -219,13 +271,17 @@ export default class AmalgamatingBusinesses extends Mixins(AmalgamationMixin, Co
   readonly BusinessLookupServices = BusinessLookupServices
   readonly EmptyBusinessLookup = EmptyBusinessLookup
 
+  // @Getter(useStore) getAmalgamatingBusinesses!: Array<AmalgamatingBusinessIF>
   @Getter(useStore) getShowErrors!: boolean
+  // @Getter(useStore) isAmalgamationFilingHorizontal!: boolean
+  // @Getter(useStore) isAmalgamationFilingRegular!: boolean
+  // @Getter(useStore) isAmalgamationFilingVertical!: boolean
```

Comment:
As in other places, these are imported via some mixins, but I like to have the declarations here anyways.